lib/bolt/cli.rb

```diff
@@ -56,7 +56,7 @@ class CLI
 HELP
 
     SCRIPT_HELP = <<-HELP.freeze
-Usage: bolt script <action> <script> [options]
+Usage: bolt script <action> <script> [[arg1] ... [argN]] [options]
```

Comment:
Is this going to result in conflicts between bolt options and script args?

Reply:
Yeah, you have to use double-dash:

```
$ bx bolt script run spec/fixtures/scripts/success.sh -n o4np54hfrljr5wo.delivery.puppetlabs.net -k -- --help
o4np54hfrljr5wo.delivery.puppetlabs.net:

arg: --help
standard out

standard error

Ran on 1 node in 0.51 seconds
```

I don't know if there is an equivalent PowerShell way of handling that (when bolt is run on Windows controller).

Reply:
Yeah, PS doesn't quite have an idea of `--` that I'm aware of... 

As of PS3, they added `--%` which instructs PowerShell itself to stop parsing, but that's not quite the same thing.

On Windows, might have to do something like quote the script arguments, which requires additional parsing... hmm...

Reply:
Oh wait, I might stand corrected.  

https://stackoverflow.com/questions/15780174/powershell-command-line-parameters-and